docs/code-quality/ca1303.md

```diff
@@ -35,14 +35,14 @@ A method passes a string literal as a parameter to a .NET constructor or method
 
 This warning is raised when a literal string is passed as a value to a parameter or property and one or more of the following cases is true:
 
-- The <xref:System.ComponentModel.LocalizableAttribute> attribute of the parameter or property is set to true.
-
-- The parameter or property name contains "Text", "Message", or "Caption".
+- The <xref:System.ComponentModel.LocalizableAttribute> attribute of the parameter or property is set to true. 
 
 - The name of the string parameter that is passed to a Console.Write or Console.WriteLine method is either "value" or "format".
 
 By default, this rule analyzes the entire codebase, but this is [configurable](#configurability).
 
+Starting with version 3 of the rule, the naming heuristic is no longer used by default but this is [configurable](#configurability).
```

Comment:
```suggestion
Starting with version 3.0.0 of the analyzer package, the naming heuristic is no longer used by default but this is [configurable](#configurability).
```

Reply:
Can we add the github link to the issue where we got quite a few customer votes on making this change in the "See also" section at the bottom of this page?